ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -337,6 +340,7 @@ const CreateRunPage = () => {
       repositoryId: Number.parseInt(params.repoId),
       requestBody: {
         revision: values.revision,
+        path: values.path,
```

Comment:
Does it make a difference for the API whether a run is created with a path of empty string or `undefined`? This would always be an empty string due to the default value, so no `undefined` or `null` can be sent.

Reply:
No, an empty string behaves exactly like providing `null`.